.github/workflows/deliver.yml

```diff
@@ -143,29 +187,46 @@ jobs:
       - build
       - sniff
       - test
+    strategy:
+      matrix:
+        platform:
+         - amd64
+         - arm32v7
+         - arm64v8
+        include:
+          - platform: amd64
+            DOCKER_PLATFORM_NAME: linux/amd64
+          - platform: arm32v7
+            DOCKER_PLATFORM_NAME: linux/arm/v7
+          - platform: arm64v8
+            DOCKER_PLATFORM_NAME: linux/arm64/v8
     steps:
       - name: Set FARMOS_VERSION from previous output.
         run: echo "FARMOS_VERSION=${{ needs.build.outputs.farmos_version }}" >> $GITHUB_ENV
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
       - name: Restore farmOS Docker image from cache
         uses: actions/cache@v3
         with:
-          path: /tmp/farmos.tar
-          key: farmos-${{ github.run_id }}
-      - name: Load farmos/farmos:3.x image
-        run: docker load < /tmp/farmos.tar
+          path: /tmp/farmos-${{ matrix.platform }}.tar
+          key: farmos-${{ matrix.platform }}-${{ github.run_id }}
+      - name: Load the Docker image
+        run: docker load < /tmp/farmos-${{ matrix.platform }}.tar
       - name: Run farmOS Docker container
-        run: docker run --rm -v /tmp/farmOS:/opt/drupal farmos/farmos:3.x true
+        run: docker run --rm -v /tmp/farmOS:/opt/drupal --platform ${{ matrix.DOCKER_PLATFORM_NAME }} $DOCKER_IMG_PREFIX:3.x-${{ matrix.platform }} true
       - name: Create artifact
-        run: cd /tmp && tar -czf farmOS-${FARMOS_VERSION}.tar.gz farmOS
+        run: cd /tmp && tar -czf farmOS-${FARMOS_VERSION}-${{ matrix.platform }}.tar.gz farmOS
       - name: Create GitHub release
         uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 #0.1.15
         with:
           body: |
             For full release notes, see [CHANGELOG.md](https://github.com/farmOS/farmOS/blob/${{ env.FARMOS_VERSION }}/CHANGELOG.md).
-          files: /tmp/farmOS-${{ env.FARMOS_VERSION }}.tar.gz
+          files: /tmp/farmOS-${{ env.FARMOS_VERSION }}-${{ matrix.platform }}.tar.gz
```

Comment:
Per the Thursday dev call, @mstenta pointed out that these releases should be the same and that we should just do these steps once.

I've confirmed that the resulting releases are identical - with the exception of the local `.git` directory which we're including and which has inconsequential differences (timestamps of checkout events I think):

e.g.

```shell
$ diff -r farmOS-3.2.3-arm32v7 farmOS-3.2.3-arm64v8
Binary files farmOS-3.2.3-arm32v7/.git/index and farmOS-3.2.3-arm64v8/.git/index differ
diff -r farmOS-3.2.3-arm32v7/.git/logs/HEAD farmOS-3.2.3-arm64v8/.git/logs/HEAD
1,3c1,3
< 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867565 +0000 clone: from https://github.com/farmOS/composer-project.git
< a7625756734acef0acf968241dc82b7ad0db4d13 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867565 +0000 checkout: moving from 3.x to 3.x
< a7625756734acef0acf968241dc82b7ad0db4d13 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867565 +0000 reset: moving to HEAD
---
> 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867590 +0000 clone: from https://github.com/farmOS/composer-project.git
> a7625756734acef0acf968241dc82b7ad0db4d13 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867590 +0000 checkout: moving from 3.x to 3.x
> a7625756734acef0acf968241dc82b7ad0db4d13 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867590 +0000 reset: moving to HEAD
diff -r farmOS-3.2.3-arm32v7/.git/logs/refs/heads/3.x farmOS-3.2.3-arm64v8/.git/logs/refs/heads/3.x
1c1
< 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867565 +0000 clone: from https://github.com/farmOS/composer-project.git
---
> 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867590 +0000 clone: from https://github.com/farmOS/composer-project.git
diff -r farmOS-3.2.3-arm32v7/.git/logs/refs/remotes/origin/HEAD farmOS-3.2.3-arm64v8/.git/logs/refs/remotes/origin/HEAD
1c1
< 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867565 +0000 clone: from https://github.com/farmOS/composer-project.git
---
> 0000000000000000000000000000000000000000 a7625756734acef0acf968241dc82b7ad0db4d13 root <root@buildkitsandbox.(none)> 1722867590 +0000 clone: from https://github.com/farmOS/composer-project.git
Binary files farmOS-3.2.3-arm32v7/web/profiles/farm/.git/index and farmOS-3.2.3-arm64v8/web/profiles/farm/.git/index differ
diff -r farmOS-3.2.3-arm32v7/web/profiles/farm/.git/logs/HEAD farmOS-3.2.3-arm64v8/web/profiles/farm/.git/logs/HEAD
1,2c1,2
< 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867648 +0000 clone: from /tmp/tmp.362UYC7utT/cache/vcs/https---github.com-symbioquine-farmOS/
< 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867651 +0000 checkout: moving from 3.x to 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf
---
> 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867686 +0000 clone: from /tmp/tmp.7KFhyI9WcC/cache/vcs/https---github.com-symbioquine-farmOS/
> 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867689 +0000 checkout: moving from 3.x to 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf
diff -r farmOS-3.2.3-arm32v7/web/profiles/farm/.git/logs/refs/heads/3.x farmOS-3.2.3-arm64v8/web/profiles/farm/.git/logs/refs/heads/3.x
1c1
< 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867648 +0000 clone: from /tmp/tmp.362UYC7utT/cache/vcs/https---github.com-symbioquine-farmOS/
---
> 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867686 +0000 clone: from /tmp/tmp.7KFhyI9WcC/cache/vcs/https---github.com-symbioquine-farmOS/
diff -r farmOS-3.2.3-arm32v7/web/profiles/farm/.git/logs/refs/remotes/origin/HEAD farmOS-3.2.3-arm64v8/web/profiles/farm/.git/logs/refs/remotes/origin/HEAD
1c1
< 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867648 +0000 clone: from /tmp/tmp.362UYC7utT/cache/vcs/https---github.com-symbioquine-farmOS/
---
> 0000000000000000000000000000000000000000 40575912ac9f7a2dc06a4f60593ef3c55bfff5bf root <root@buildkitsandbox.(none)> 1722867686 +0000 clone: from /tmp/tmp.7KFhyI9WcC/cache/vcs/https---github.com-symbioquine-farmOS/
diff -r farmOS-3.2.3-arm32v7/web/profiles/farm/.git/objects/info/packs farmOS-3.2.3-arm64v8/web/profiles/farm/.git/objects/info/packs
1c1
< P pack-6947608a5509b77434cf0d3b7acf81f4e9d75744.pack
---
> P pack-1a2be076db4ddbb5c5cccfdb4265a0b86a5bab27.pack
Only in farmOS-3.2.3-arm64v8/web/profiles/farm/.git/objects/pack: pack-1a2be076db4ddbb5c5cccfdb4265a0b86a5bab27.idx
Only in farmOS-3.2.3-arm64v8/web/profiles/farm/.git/objects/pack: pack-1a2be076db4ddbb5c5cccfdb4265a0b86a5bab27.pack
Only in farmOS-3.2.3-arm32v7/web/profiles/farm/.git/objects/pack: pack-6947608a5509b77434cf0d3b7acf81f4e9d75744.idx
Only in farmOS-3.2.3-arm32v7/web/profiles/farm/.git/objects/pack: pack-6947608a5509b77434cf0d3b7acf81f4e9d75744.pack
```